js/private/node_package.bzl

```diff
@@ -12,13 +12,12 @@ _NodePackageInfo = provider(
         "package": "name of this node package",
         "version": "version of this node package",
         "dep_refs": "list of dependency ref targets",
+        "bins": "A dict of bin names to paths for this package",
```

Comment:
Is this for adding additional bin paths for the lifecycle hook? Not sure if you need this, but I believe there is an option in the lifecycle hook library I was using for this, or if I recall correctly it might be buried in the library beneath and passed through `rawConfig`.

Reply:
It ends up generating the `node_modules/.bin/tool` entry points which are needed for lifecycle hooks to work.

There is an `extraBinPaths` but that is not needed.